core-common/src/main/java/org/zalando/nakadi/domain/kpi/AccessLogEvent.java

```diff
@@ -1,8 +1,27 @@
 package org.zalando.nakadi.domain.kpi;
 
+import org.apache.avro.Schema;
 import org.zalando.nakadi.config.KPIEventTypes;
+import org.zalando.nakadi.util.AvroUtils;
+
+import java.io.IOException;
 
 public class AccessLogEvent extends KPIEvent {
+
+    private static final String PATH_SCHEMA = "event-type-schema/nakadi.access.log/nakadi.access.log.2.avsc";
+    private static final Schema SCHEMA;
+
+    static {
+        // load latest local schema
+        try {
+            SCHEMA = AvroUtils.getParsedSchema(
```

Comment:
Can we move the static block to `KPIEvent` class as a static method instead of duplicating it, then we can initialize the `SCHEMA` in one line by calling it.